cmd/osv-scanner/main_test.go

```diff
@@ -62,7 +62,25 @@ func normalizeRootDirectory(t *testing.T, str string) string {
 	return strings.ReplaceAll(str, cwd, "<rootdir>")
 }
 
-// normalizeRootDirectory attempts to replace references to the temp directory
+// normalizeUserCacheDirectory attempts to replace references to the current working
+// directory with "<tempdir>", in order to reduce the noise of the cmp diff
```

Comment:
```suggestion
// normalizeUserCacheDirectory attempts to replace references to the users cache
// directory with "<tempdir>", in order to reduce the noise of the cmp diff
```

Reply:
note, I think it's fine now but in future we might want to switch to explicitly referring to this as `cachedir` and having a conditional normalize when the `--experimental-local-db` flag is present to replace `cachedir` with `tempdir`, but that seems pretty overkill to me for now

Reply:
yeah this is my thoughts too its not pretty but itll do for now